pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -557,6 +580,15 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		dpiComponent,
 		rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
 			Namespace:       render.IntrusionDetectionNamespace,
+			ServiceAccounts: []string{render.IntrusionDetectionName},
+			KeyPairOptions: []rcertificatemanagement.KeyPairOption{
+				rcertificatemanagement.NewKeyPairOption(intrusionDetectionCfg.IntrusionDetectionCertSecret, true, true),
+			},
+			TrustedBundle: trustedBundle,
+		}),
+		rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
+			Namespace: render.IntrusionDetectionNamespace,
+			// TODO: ALINA - CAN WE REMOVE THIS SERVICE ACCOUNT AND LEAVE ONLY AD API ?
```

Comment:
I'm not sure, to be honest. I don't know the details of this component that well. 

We might be able to combine both of these CertificateManagement compoinents down into one, though.